lib/fs_utils/generate.js

```diff
@@ -188,6 +188,11 @@ const optimize = (data, map, path, optimizers, sourceFiles) => {
   return promiseReduce(optimizers, runOptimizer, initial);
 };
 
+const getInlineSourceMapUrl = map => {
+  const base64String = new Buffer(JSON.stringify(map)).toString('base64');
```

Comment:
Brunch claims to support Node `>= 4`, and `Buffer.from` was only added in `5.10`. So, for backwards compatibility, should I do this?

```js
const base64String = (Buffer.from ? Buffer.from(JSON.stringify(map)) : new Buffer(JSON.stringify(map))).toString('base64');
```